### ISSUE TITLE
Bandsaw needs a new "tyre"

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#282]
One of the wheels that the bandsaw blade goes round had a solid rubber tyre so the blade gripped better.

It had perished.  Rob found the problem when looking into why it wasn't working well, and removed the tyre.  However, there's now too much play in the blade for it to work well, so we've taken it out of action.

Need to work out if new tyres are available for it, and get one ordered and fitted.

**Comment by jamieisboss** [comment]
[Any of these do?](http://www.bedfordsaw.co.uk/spare-parts)

**Comment by amcewen** [comment]
I don't know.  Not obviously.  See https://github.com/DoESLiverpool/wiki/wiki/Bandsaw for details of the bandsaw in question.

**Comment by jamieisboss** [comment]
[Only hit in my google result
](http://www.ebay.co.uk/itm/2-BLUE-MAX-HEAVY-DUTY-URETHANE-BAND-SAW-TIRES-REPLACES-FERM-PART-403461-/261199481543)

**Comment by jamieisboss** [comment]
@amcewen seems like the only other one is dearer and not as good, it is just a big rubber band though, though you can buy a new saw for double that price

**Comment by goatchurchprime** [comment]
Quite expensive for what it is, incl postage from America.  Might have to bite the bullet any.

http://www.ebay.co.uk/itm/2-BLUE-MAX-ULTRA-DUTY-URETHANE-BAND-SAW-TIRES-FOR-FERM-FLZ-275-BAND-SAW-/271648557722

http://www.ebay.co.uk/itm/FERM-403461-BLUE-MAX-HEAVY-DUTY-BAND-SAW-TIRES-REPLACES-FERM-403461-TYRES-/262653044166?hash=item3d275765c6

**Comment by jackie1050** [comment]
I've emailed Ferm (although part 403461 seems to be specifically missing from the list here http://service.ferm.com/en_service/bsm4007-flz-275-band-saw.html).
Also emailed Axminster (tools supplier recommended by my dad).
Maybe only Americans use them.

**Comment by drfraz** [comment]
I had a look around a few tool suppliers in Liverpool and none stocked
them. I suspect any 20 cm/8' diameter tire would work, most of them seem to
be about £30 on ebay for the urethane ones

**Comment by amcewen** [comment]
I guess we can see what Ferm say when/if they get back to @jackie1050, but look to get just a generic 20cm tire to try if they don't offer anything better.

**Comment by amcewen** [comment]
While looking at the manufacturer's page that @jackie1050 linked to I noticed the manual, so created a [Manuals area in our "somewhere-safe" repo](https://github.com/DoESLiverpool/somewhere-safe/tree/master/Workshop/Manuals) and stashed a copy of it there for safe-keeping.

**Comment by stevesparrow** [comment]
Axminister do god quality stuff. Just for reference it's where I bought the
new blades.

S

---

**Comment by jackie1050** [comment]
Axminster sell a 14mm wide tyre to fit a 200mm wheel - order code CCB-6 £9.88 - it can be ordered through their sales line on 0800 371822

**Comment by DoESsean** [comment]
I've ordered this from Axminster.

We've ordered two, as the suggestion is that you replace the tyres on both the top and bottom wheels at the same time, as the rate of wear is going to be the same. They're expected tomorrow (Friday 27/1), but should be here by Monday at the latest.

**Comment by DoESsean** [comment]
Tyres have arrived from Axminster. I've put them on the saw ready for fitting.

**Comment by drfraz** [comment]
Tyre replaced & tracking adjusted. Seems to run & cut OK
Spare tyre in draw next to spare blade

**Comment by Dougk10** [comment]
I've had the same problem, and ordered tyres from Axminster.  They won't be in stock for at least a couple of months, so I did a DIY repair.  I got some 1/2" * 1/8" rubber strip from ebay, and a tube of Evostick.  The wheel was a bit gummy, so I sanded the tyre groove until it was bright.  I also roughed up the surface of the rubber and cut it to length.  It glued on easily, except the join of the ends, which was a bit fiddly, but ended up really well.  It works perfectly.  If you have one of these machines I would recommend buying a pair of tyres before they disintigrate.
